libparsec/crates/platform_mountpoint/Cargo.toml

```diff
@@ -17,7 +17,12 @@ regex = { workspace = true }
 winfsp_wrs = { workspace = true }
 
 [target.'cfg(target_family = "unix")'.dependencies]
-fuser = { workspace = true, features = ["libfuse", "abi-7-21"] }
+# ABI 7.31 has been introduced in Linux 5.2 (2019-06-08), should be old enough
```

Comment:
Shouldn't this be carefully tested?

A quick search shows that Ubuntu `22.10` (and older) started with initial kernel version _older_ than `5.2`. On Debian, only `12` (stable) started with a `6.x` version. 
Most up-to-date systems will meet the requirement, but I wouldn't be surprised if the administration still uses "old" RHEL/CentOS laptops. Maybe we should check that?

Reply:
I wouldn't be concerned about this: if your kernel is still from 2019 that means your have never updated your system... [then you have much bigger issues than not being able to run Parsec :trollface: ](https://www.cvedetails.com/product/47/Linux-Linux-Kernel.html?vendor_id=33)